lib/inspec/base_cli.rb

```diff
@@ -80,74 +80,97 @@ def self.exec_options
     def self.default_options
       {
         exec: {
-          'reporter' => {
-            'cli' => nil,
-          },
+          'reporter' => ['cli'],
           'show_progress' => false,
           'color' => true,
           'create_lockfile' => true,
           'backend_cache' => false,
         },
+        shell: {
+          'reporter' => ['cli'],
+        },
       }
     end
 
-    def self.clean_reporters(opts)
-      # Parse report options from CLI
+    def self.parse_reporters(opts)
+      # merge in any legacy formats as reporter
+      # this method will only be used for ad-hoc runners
+      if !opts['format'].nil? && opts['reporter'].nil?
+        warn '[DEPRECATED] The option --format is being is being deprecated and will be removed in inspec 3.0. Please use --reporter'
+        opts['reporter'] = [opts['format']]
```

Comment:
If `opts['format']` has the chance of being an Array, this would do an array-inside-an-array which may not be what you intended.

If you just want to ensure that it's always an array, it's generally safer to do:

```ruby
opts['reporter'] = Array(opts['format'])
```

It's probably safe in this case since this gets populated via Thor and we control the type there, but something to keep in mind for the future.